app/views/lessons/show.html.erb

```diff
@@ -1,40 +1,39 @@
 <%= title(@lesson.title) %>
 
-<!-- progress-tracking bar -->
-<%= render 'shared/pt_bar', lesson: @lesson %>
-
-<div class="individual-lesson <%= @lesson.is_project ? 'project-lesson' : ''%>">
+<div class="individual-lesson">
   <div class="container">
+    <div class="row">
+      <div class="col-lg-10 offset-lg-1">
+        <%= render 'courses/course/banner', course: @course, show_completion_percent: true %>
```

Comment:
I'm not sure if we will be using the percentage completed component on lesson pages. It isn't in the mock ups at least. But lets leave it to see how it looks and functions.